dls-controls/malcolmjs
# Feature/update to source sink

## Description

Updated the in/out ports to source/sink. @btester271828 I don't understand why the canned data is still in here as I've just merged in the canned data PR. You might just have to view it by the last commit below.

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- Start the site and check the layout still renders properly.

## Agile board tracking
